static/js/filter.js

```diff
@@ -1,7 +1,95 @@
 (function() {
 	"use strict";
-
     
-	
-	
-})();
\ No newline at end of file
+	const filterEmailAddresses = [];
+	const filterAliasLabels = [];
+	const aliasesWithLabelsCollection = [];
+    const aliasCollection = [];
+    const aliases = document.querySelectorAll(".c-alias");
+
+    const filterLabelTotalCases = document.querySelector(".js-filter-case-total");
+    const filterLabelVisibleCases = document.querySelector(".js-filter-case-visible");
+
+    const filterResetButton = document.querySelector(".js-filter-reset")
+
+    const filterInput = document.querySelector(".js-filter-search-input");
```

Comment:
Looks like these are all single elements, so these could change from class names to element IDs and the JS could be `document.getElementById` ?

Reply:
I'm wanting to use the `js-` naming convention to make it easy to identify JavaScript enabled items in the code. This may need a larger conversation, bc I agree with you on leveraging `getElementById` over `querySelector`.